src/api/endpoints/logged_activities/crud.py

```diff
@@ -72,6 +74,20 @@ def post(self):
                     ]
 
             logged_activity.save()
+            society_id = g.current_user.society_id
+
+            #send notification to the society secretary about logged points
+
+            roles = User.query.filter(User.roles.any(Role.name=="society secretary")).all()
+            users = User.query.filter_by(society_id=society_id).all()
```

Comment:
We could also make users a queryset stored as a class variable, then only filter in the POST call (user.filter_by...)

Reply:
Hmm. I'm thinking of it from a Django perspective. I'll look more into it from a Flask/SQLAlchemy perspective, but we can go with your current implementation for now.